app/src/components/domain/DrefExportModal/index.tsx

```diff
@@ -117,16 +212,52 @@ function DrefExportModal(props: Props) {
             {(exportStatusResponse?.status === EXPORT_STATUS_ERRORED
                 || isDefined(exportTriggerError)
                 || isDefined(exportStatusError)
+                || isDefined(drefImminentExportError)
             ) && (
                 <Message
                     title={strings.drefExportFailed}
                     description={exportTriggerError?.value.messageForNotification
-                            ?? exportStatusError?.value.messageForNotification}
+                            ?? exportStatusError?.value.messageForNotification
+                            ?? drefImminentExportError?.value.messageForNotification}
                 />
             )}
+            {!(pendingExportTrigger
+                || pendingExportStatus
+                || exportStatusResponse?.status === EXPORT_STATUS_PENDING)
+                && drefType === DREF_TYPE_IMMINENT
+                && !drefImminentExportError && (
+                <div className={styles.downloadButton}>
```

Comment:
Both `Message` and `Button` probably should not be wrapped by `div` with `className={styles.downloadButton}`